### PR TITLE
Support QRDA Cat III R2.1 export

### PR DESCRIPTION
**Pull request by dczulada**
Add QRDA Cat III R2.1 schematron validator

### CODE REVIEW

**Review by tstrass — APPROVED** [review]

**Review by edeyoung — APPROVED** [review]
I have not had a chance to check the QRDA or have someone on my team do it. However, since this is related to cat_3 and the rest of the logic looks sound to me, I approve.

File: templates/cat3/r2_1/_measure_data.cat3.erb [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      <value xsi:type="INT" value="<%= population.value.round %>"/>
+      <methodCode code="COUNT" 
+        displayName="Count" 
+        codeSystem="2.16.840.1.113883.5.84" 
+        codeSystemName="ObservationMethod"/>
+    </observation>
+  </entryRelationship>
+  <% population.stratifications.each do |strat| -%>
+
+  <!--  Startification Reporting Template for <%= population.type %>  <%= population.id %>  Stratification <%= strat.id %>   -->
```
Comment (okeefm) [inline templates/cat3/r2_1/_measure_data.cat3.erb]: `Startification` should be `Stratification`

File: templates/cat3/r2_1/_measure_data.cat3.erb [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                :template_ids => ['2.16.840.1.113883.10.20.27.3.8'], :extension => '2016-09-01',  :supplemental_data_code => '72826-1',
+                :supplemental_data_code_system => '2.16.840.1.113883.6.1', :supplemental_data_value_code => race,
+                :supplemental_data_value_code_system => '2.16.840.1.113883.6.238', :count => count, :qrda3_version => qrda3_version} %>   
+      <% end -%>
+    <% end -%>
+  <%   payer_supplimental_data = population.supplemental_data["PAYER"]
+       if payer_supplimental_data.present?
+         payer_supplimental_data.each do |payer, count| -%>
+
+ <!--         PAYER Supplemental Data Reporting   for<%= population.type %>  <%= population.id %>   -->
```
Comment (okeefm) [inline templates/cat3/r2_1/_measure_data.cat3.erb]: Can you add a space between `Data reporting for` and `<%= population.type %>`

File: templates/cat3/r2_1/_supplemental_data.cat3.erb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+  <!--                            Supplemental Date Template                                                  -->
```
Comment (okeefm) [inline templates/cat3/r2_1/_supplemental_data.cat3.erb]: This should say `supplemental Data` not `Supplemental Date`

**Review by okeefm — CHANGES_REQUESTED** [review]

**Review by okeefm — APPROVED** [review]

**Pull request merged**